.eslintrc

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": [
+        "@aerian/eslint-config-react",
+        "@aerian/eslint-config-typescript"
+    ],
+    "rules": {
+        "jsx-a11y/heading-has-content": 0,
+        "complexity": ["warn", 6]
```

Comment:
This is now the default

Reply:
That was restrained of you! I imagined you'd have put it up to 8.